File: Examples/CSharp/Articles/AddXmlMapInsideWorkbook.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class AddXmlMapInsideWorkbook
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Create workbook object
            Workbook wb = new Workbook();

            // Add xml map found inside the sample.xml inside the workbook
            wb.Worksheets.XmlMaps.Add(sourceDir + "sampleAddXmlMapInsideWorkbook.xml");

            // Save the workbook in xlsx format
            wb.Save(outputDir + "outputAddXmlMapInsideWorkbook.xlsx");

            Console.WriteLine("AddXmlMapInsideWorkbook executed successfully.\r\n");
        }
    }
}

File: Examples/VisualBasic/Articles/AddXmlMapInsideWorkbook.vb
Namespace Articles
    Public Class AddXmlMapInsideWorkbook
        Public Shared Sub Run()
            ' ExStart:AddXmlMapInsideWorkbook
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Create workbook object
            Dim wb As New Workbook()

            ' Add xml map found inside the sample.xml inside the workbook
            wb.Worksheets.XmlMaps.Add(dataDir & Convert.ToString("sample.xml"))

            ' Save the workbook in xlsx format
            wb.Save(dataDir & Convert.ToString("output_out.xlsx"))
            ' ExEnd:AddXmlMapInsideWorkbook
        End Sub
    End Class
End Namespace